core/tests/protractor_desktop/fileUploadExtensions.js

```diff
@@ -0,0 +1,76 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview End-to-end tests for rich-text components involving file
+ * upload.
+ */
+
+var forms = require('../protractor_utils/forms.js');
+var general = require('../protractor_utils/general.js');
+var users = require('../protractor_utils/users.js');
+var workflow = require('../protractor_utils/workflow.js');
+var waitFor = require('../protractor_utils/waitFor.js');
+
+var ExplorationEditorPage =
+  require('../protractor_utils/ExplorationEditorPage.js');
+var ExplorationPlayerPage =
+  require('../protractor_utils/ExplorationPlayerPage.js');
+
+describe('rich-text components', function() {
+  var explorationEditorPage = null;
+  var explorationEditorMainTab = null;
+  var explorationPlayerPage = null;
+
+  beforeEach(function() {
+    explorationEditorPage = new ExplorationEditorPage.ExplorationEditorPage();
+    explorationEditorMainTab = explorationEditorPage.getMainTab();
+    explorationPlayerPage = new ExplorationPlayerPage.ExplorationPlayerPage();
+  });
+
+  it('should display math expressions correctly', async function() {
+    await users.createUser(
+      'user@richTextComponents.com', 'userRichTextComponents');
+    await users.login('user@richTextComponents.com');
```

Comment:
This username could conflict with other test suites, namely:

https://github.com/oppia/oppia/blob/21d345c5ce055af4b2aa5625927b06eac3ed565b/core/tests/protractor_desktop/extensions.js#L44-L45

Maybe `richTextuser@fileUploadExtensions.com`?

Reply:
done